src/oaklib/cli.py

```diff
@@ -207,6 +211,7 @@
     JSON_FORMAT,
     YAML_FORMAT,
     FHIR_JSON_FORMAT,
+    FHIR_NPM_FORMAT,
```

Comment:
@cmungall I haven't actually properly set up the CLI yet. It can call the dumper for `fhirnpm`, but need to settle on the params for it first, and then probably document the config for that as with the `fhirjson` dumper.

Reply:
@cmungall You may consider this a blocker, the fact that there's no CLI. Since I didn't do it back in April, I'm thinking it's because I wanted your input on the params for some reason. If this is needed for merge, I can look into it and let you know if/where I need feedback.